src/state.rs

```diff
@@ -0,0 +1,30 @@
+pub(crate) mod game;
+
+use notan::prelude::*;
+
+use game::{defeat, playing, victory, Stage};
+
+#[derive(AppState)]
+pub struct State {
```

Comment:
I also think, for future split,  this should be called GameState

Reply:
This is supposed to be the `GlobalState`, and I'd like to avoid adding hungarian-like prefixes to struct names, that's what modules and aliases are for